schemas/io-package.json

```diff
@@ -0,0 +1,457 @@
+                        "visualization-widgets", "weather"]
+                },
+                "unsafePerm": {
+                    "description": "if the package must be installed with \"npm --unsafe-perm\" parameter",
+                    "type": "boolean"
+                },
+                "plugins": {
+                    "description": "ioBroker plugins to be used",
+                    "type": "object"
+                },
+                "adminColumns": {
+                    "description": "Custom attributes, that must be shown in admin in the object browser. Like: [{\"name\": {\"en\": \"KNX address\"}, \"path\": \"native.address\", \"width\": 100, \"align\": \"left\"}, {\"name\": \"DPT\", \"path\": \"native.dpt\", \"width\": 100, \"align\": \"right\", \"type\": \"number\", \"edit\": true, \"objTypes\": [\"state\", \"channel\"]}]. type is a type of the attribute (e.g. string, number, boolean) and only needed if edit is enabled. objTypes is a list of the object types, that could have such attribute. Used only in edit mode too.",
+                    "type": "array"
+                },
+                "adminTab": {
+                    "description": "Adds a Tab which can be shown in admin adapter",
+                    "type": "object",
+                    "required": ["fa-icon", "link", "name"],
+                    "additionalProperties": false,
+                    "properties": {
+                        "fa-icon": {
+                            "description": "Font-Awesome icon name for TAB",
+                            "type": "string"
+                        },
+                        "ignoreConfigUpdate": {
+                            "description": "Do not update config TAB if configuration changed (to enable configure settings in TAB)",
+                            "type": "boolean"
+                        },
+                        "link": {
+                            "description": "Link for iframe in the TAB. You can use parameters replacement like this: \"http://%ip%:%port%\". IP will be replaced with host IP. \"port\" will be extracted from native.port.",
+                            "type": "string"
+                        },
+                        "name": {
+                            "description": "Name of TAB (multilingual object) in admin",
+                            "type": "object",
+                            "required": ["en"],
+                            "patternProperties": {
+                                "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+                                    "type": "string"
+                                }
+                            },
+                            "additionalProperties": false
+                        },
+                        "singleton": {
+                            "description": "If true, only one TAB for all instances will be shown.",
+                            "type": "boolean"
+                        }
```

Comment:
Should we allow `false` for these properties that are only useful if set to `true`?
==> `"const": true` would be the alternative

Reply:
Hm doesn't really hurt if they are there, so I would allow it if someone wants to have it for overview reasons